.github/workflows/pull_request.yml

```diff
@@ -106,6 +106,11 @@ jobs:
       - name: Setup Rust & Cargo
         uses: ./.github/actions/setup_rust_cargo
 
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.1"
+          working-directory: context-ruby
+
```

Comment:
Waiting for custom image from our self-hosted runners for ruby to work on self-hosted. This means that the trunk check step will fail until updated.